sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -146,7 +146,7 @@ public async Task DeleteSetting()
             }
             finally
             {
-                await service.DeleteConfigurationSettingAsync(testSetting.Key, testSetting.Label);
+                AssertStatus200(await service.DeleteConfigurationSettingAsync(testSetting.Key, testSetting.Label));
```

Comment:
Why doesn't this fail? We are deleting the secret at https://github.com/Azure/azure-sdk-for-net/pull/11205/files#diff-3aa297bb699ae547859d1b6af28fb9dbR137

Reply:
`testSettingDiff` has no label.